CharlieBackend.AdminPanel/Exceptions/HttpStatusException.cs

```diff
@@ -0,0 +1,26 @@
+﻿using CharlieBackend.Core.DTO.Result;
+using Newtonsoft.Json;
+using System;
+using System.Net;
+using System.Net.Http;
+
+namespace CharlieBackend.AdminPanel.Exceptions
+{
+    class HttpStatusException : Exception
+    {
+        private static string GetFormattedMessage(HttpResponseMessage httpResponseMessage)
+        {
+            string stringResponse = httpResponseMessage.Content.ReadAsStringAsync().GetAwaiter().GetResult();
+            var message = $"{JsonConvert.DeserializeObject<ErrorDto>(stringResponse).Error.Message}";
```

Comment:
Why do you need string interpolation here?

Reply:
Sorry, I forgot to delete this